lib/support/cli.js

```diff
@@ -283,6 +283,11 @@ module.exports.parseCommandLine = function parseCommandLine() {
         'To add multiple arguments, repeat --firefox.android.intentArgument once per argument.',
       group: 'firefox'
     })
+    .option('firefox.profileTemplate', {
+      describe:
+        'Profile template directory',
```

Comment:
This might need some more info on exactly what this is.  Is it a preconditioned profile?

Reply:
Yes, it can be.  I originally added it to allow to inject MITM certs into the Firefox profiles used, but it can be a completely conditioned profile just as well.

I've expanded the description, which was just lazy/expedient.